homeworks/kostiantyn.nosatskyi_NosatskyiK/PokemonProject/style.css

```diff
@@ -0,0 +1,28 @@
+h1 {
+  font-size: 48px ;
+  font-family: Roboto, serif ;
```

Comment:
[Roboto is a sans-serif font](https://en.wikipedia.org/wiki/Roboto#:~:text=Roboto%20(%2Fro%CA%8A%CB%88b,4.0%20%22Ice%20Cream%20Sandwich%22.), so if you like to state here a generic default, please do it in the right manner
![image](https://github.com/MastersAcademy/fe-course-basic-2023/assets/98031768/275ce146-92bf-4bec-b9ea-d50781aa7704)